src/data/accessibility.json

```diff
@@ -0,0 +1,55 @@
+            "We are working to remove the listed barriers as soon as possible.",
+            "<b>Date of creation or last update of the declaration</b>",
+            "This statement was created on Feb 21, 2022."
+        ]
+    },
+    "section-report-barriers": {
+        "headline": {
+        "title": "Report barriers: Feedback contacts",
+        "text": "You would like to inform us about existing barriers or request information on the implementation of accessibility? You are welcome to contact us by <a href='coronawarnapp...rki...de' class='email'>e-mail</a> or via our <a href='https://www.rki.de/SharedDocs/Kontaktformulare/weitere/Corona-Warn-App/Corona-Warn-App_Integrator.html' target='_blank' rel='noopener noreferrer'>contact form</a>. The contact form is only available in German."
+        }
+    },
+    "section-arbitrary-service": {
+        "title": "Arbitration",
+        "textblock": [
+            "In case no satisfactory solution has been found even after your feedback to the above-mentioned contact, you can approach the arbitration service in accordance with § 16 BGG. The BGG arbitration service is responsible for supporting out-of-court dispute resolution in the event of conflicts on the topic of accessibility between people with disabilities and federal public entities. The arbitration service is free of charge. No legal counsel needs to be involved. For more information on the arbitration service and how to submit an application, please visit the <a href='https://www.schlichtungsstelle-bgg.de/Webs/SchliBGG/EN/home' target='_blank' rel='noopener noreferrer'>website of the arbitration service</a>.",
+            "You can reach the arbitration service at the following address:",
+            "BGG Arbitration Service<br/>Beauftragter der Bundesregierung für die Belange von Menschen mit Behinderungen<br/>Mauerstrasse 53<br/>10117 Berlin<br/>Telephone: +49 (0)30 18 527-2805<br/>Fax: +49 (0)30 18 527-2901<br/>E-Mail: <a href='info...schlichtungsstelle-bgg...de' class='email'>info@schlichtungsstelle-bgg.de<a/><br/>Internet: <a href='https://www.schlichtungsstelle-bgg.de/Webs/SchliBGG/EN/home' target='_blank' rel='noopener noreferrer'>https://www.schlichtungsstelle-bgg.de/Webs/SchliBGG/EN/home</a>"
```

Comment:
The phone numbers are not formatted according to international standard [E.123](https://en.wikipedia.org/wiki/E.123) which says:

"Parentheses are not allowed in the international notation, according to the standard, as international callers use fixed number dialing."

So "Telephone: +49 (0)30 18 527-2805<br/>Fax: +49 (0)30 18 527-2901" =>
"Telephone: +49 30 18 527-2805<br/>Fax: +49 30 18 527-2901"

This also conforms to the way other international phone numbers are rendered on coronawarn.app.

Reply:
Fixed in https://github.com/corona-warn-app/cwa-website/pull/2508/commits/8b0a4178807bc2a0c8f46aafdf4f47e25b2440fb, thanks!